File: C++/Day-1/Lab-1-Google Drive/ReverseNumber.cpp
//15: Write a  program to enter a number and print its reverse.

#include<iostream>

using namespace std;

int reverseNumber(int number){
	int reverse = 0;
	
	while(number > 0){
		
		int rem = number % 10;
		reverse = reverse *10 + rem;
		number = number/10;
	}
	
	return reverse;
}

int main(){
	
	int number;
	
	cout<<"Enter the number = ";
	cin>>number;
	
	int reverseNum = reverseNumber(number);
	
	cout<<"The reverse of number "<<number<<" is : "<<reverseNum;
	
	return 0;
	
}


	
	
	

File: JAVA/NumberSeries/src/ReverseNumber/ReverseNumber.java
package ReverseNumber;

public class ReverseNumber {

    public static void main(String[] args) {
        int number = 12345234;
        int rem;

        int revNum = 0;

        while (number != 0) {
            rem = number % 10;

            revNum = revNum*10 + rem;

            number = number / 10;
        }

        System.out.println(revNum);
    }
}
